cmd/query/app/server.go

```diff
@@ -219,6 +222,12 @@ func createHTTPServer(
 			MeterProvider:  telset.MeterProvider,
 		},
 		handler,
+		xconfighttp.WithOtelHTTPOptions(
+			otelhttp.WithFilter(func(r *http.Request) bool {
+				ignorePaths := regexp.MustCompile(`^/static/.*`)
```

Comment:
1. `^/static/.*` does not require regex, can be matched with `strings.HasPrefix`
2. it doesn't look like it will work if the UI is running with a custom base path, we need to account for that too

Reply:
@yurishkuro for 2 - do we want to do a `contains` to see if the `/static` path is anywhere in the URL?

Reply:
that would be a bit fuzzy. I would rather use HasPrefix with `basePath + "/static/"`

Reply:
done

Reply:
```diff
git diff cmd/jaeger/config.yaml

   jaeger_query:
+    base_path: /foobar
     storage:
       traces: some_store
       traces_archive: another_store
```
```
$ go run ./cmd/jaeger --config ./cmd/jaeger/config.yaml
```
Reload UI a few times, observe span `/foobar/static/favicon-BxcVf0am.ico`

So, not solved.

Reply:
@yurishkuro apologies - i forgot to push my commits